shared/search/user-input/index.native.js

```diff
@@ -203,7 +204,7 @@ class UserInput extends Component<Props, State> {
             }}
           >
             <TextInput
-              autoFocus={autoFocus}
+              autoFocus={flags.useNewRouter ? false : autoFocus}
```

Comment:
Why this change?

Reply:
this can likely go away, i was having issues w/ the keyboard avoiding and how it interacted w/ the animating layers. i suspected it had to do with the focusing coming / going but i think it was due to something else